packages/react-onegraph/src/index.js

```diff
@@ -1,90 +1,97 @@
-import React, {Component, createContext} from 'react';
-import OneGraphAuth from 'onegraph-auth';
+import React, {Component, createContext} from 'react'
```

Comment:
Can you run `yarn prettier`. That should remove the formatting changes in the diff.